dkc/core/migrations/0009_file_pre_creation.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.6 on 2021-02-03 15:40
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('core', '0008_quota'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='file',
+            name='size',
+            field=models.PositiveBigIntegerField(),
+        ),
+        migrations.AlterField(
+            model_name='quota',
+            name='user',
+            field=models.OneToOneField(
+                on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL
+            ),
```

Comment:
@brianhelba this seems to have slipped in from a change on a prior branch that wasn't captured in a migration.

Reply:
Yeah, I'll add something to test for that in the future.

Reply:
https://github.com/girder/cookiecutter-django-girder/pull/125 will ensure this doesn't happen again, once we pull it down.